ui/app/templates/components/oidc/assignment-form.hbs

```diff
@@ -42,33 +42,29 @@
         <AlertInline @type="danger" @message={{join ", " this.modelValidations.name.errors}} />
       {{/if}}
       {{#if (or @model.canListEntities @model.canListGroups)}}
```

Comment:
Should we instead wrap each of the search-selects with their respective permissions? Or leave it out since it will render a fallback anyway?

Reply:
I think Angel did this because an assignment has to have either a group or an entity. I think there may have been a discussion with design about it - but I'll follow up with her when she's back! 

That's a good point about the fallback 🤔

Reply:
Yep, this was a design decision. When they don't have access to one, the drop down is empty. But if they don't have access to both then we show the permissions message. It felt better than showing a permissions warning message above or below another search select when really it only matters if they don't have permissions to both.